smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -45,16 +51,15 @@ def authPage() {
 		// This is the SmartThings access token
 		if (!state.accessToken) {
 			log.debug "no access token, create access token"
-			createAccessToken() // predefined method
+			state.accessToken = createAccessToken() // predefined method
 		}
 		def description = "Tap to enter LIFX credentials"
-		def redirectUrl = "${serverUrl}/oauth/initialize?appId=${app.id}&access_token=${state.accessToken}" // this triggers oauthInit() below
+		def redirectUrl = "${serverUrl}/oauth/initialize?appId=${app.id}&access_token=${state.accessToken}&apiServerUrl=${shardUrl}" // this triggers oauthInit() below
```

Comment:
We should probably switch `shardUrl` to `apiServerUrl`.  Its a little more clear since shardUrl is a bit of an overloaded term.